salesforce/campaign.py

```diff
@@ -0,0 +1,54 @@
+from .client import SalesforceClient
+import json
+import requests
+import threading
+''' Project model maps to Salesforce Campaign object '''
+client = SalesforceClient()
+
+
+def run(request):
+    response = SalesforceClient().send(request)
+
+
+def save(project: object):
+    stage_tags = list(project.project_stage.all().values())
```

Comment:
In order to retrieve a Tag's display name from its slug, first retrieve the Tag object with
https://github.com/DemocracyLab/CivicTechExchange/blob/7a6588ae5417b655e58a448ff023b3523d418345/common/models/tags.py#L23

Once you have the Tag object, you can access the Tag's display name with
https://github.com/DemocracyLab/CivicTechExchange/blob/7a6588ae5417b655e58a448ff023b3523d418345/common/models/tags.py#L7